src/index.html

```diff
@@ -17,6 +29,85 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a
+        href=" "
```

Comment:
The `href` attribute is empty. Consider providing a valid URL or a placeholder (`#`) to ensure the link is functional.